### DISCUSSION TITLE
Pre-backup script?

### DISCUSSION THREAD

**Discussion by beastarman74** [tiredofit/docker-db-backup#148] [Q&A]
Hi,
Is there the option of a pre-backup script.  It looks like one of my containers (uptime-kuma) requires the container to be stopped before it will allow a backup to be taken.

When running the backup throws a "file is not a database" error.

Thanks.

**Comment by tiredofit** [comment]
Hi there, I added options for pre backup scripts in `3.4.0`.  Please read the README for the location to load Pre scripts from and let me know if you encounter any issues.
